### ISSUE TITLE
Rename distribution from `classic` to `classic-ui` (preferred) or `classicui`

### ISSUE THREAD

**Issue by stevepiercy** [plone/plone.classicui#18]
`classic` is not its correct name. `classic-ui` (preferred) or `classicui` would be more correct.

See https://github.com/plone/documentation/pull/1733#discussion_r1827159965.

Edit https://github.com/plone/plone.classicui/blob/main/src/plone/classicui/distributions.zcml#L7.

**Comment by davisagli** [comment]
Personally, unlike the volto distribution, I don't think this one is worth the effort to change. It's mostly an internal identifier. (I would agree with removing it from the launch page.) And a distribution encompasses more than just frontend settings, so it's sort of correct to not include UI in the name.

**Comment by stevepiercy** [comment]
Maybe we call it `base` or `core` instead of `classic`?

What else goes into this distribution? It sounds like it is only those pieces that are required to run Plone, plus Classic UI bits. Looking into the future and reading the road map for Plone 7, it sounds like this distribution will become Plone core only, and a new distribution called `classic-ui` would be `base` + Classic UI bits and pieces.

Finally, when people say "classic", they mean Classic UI.

**Comment by mauritsvanrees** [comment]
No, in Plone 7 the classic distribution will be roughly the same as now. It will keep pulling in or activating code that CMFPlone 7 no longer does. For example it would still pull in the Barceloneta theme, where CMFPlone 7 might not. 
No base distribution is planned. Not by me anyway. 😁

**Comment by davisagli** [comment]
> Finally, when people say "classic", they mean Classic UI.

I don't understand why you see these in conflict. The distribution named `classic` is the one which sets up a Plone site with the Classic UI.

**Comment by stevepiercy** [comment]
> The distribution named `classic` is the one which sets up a Plone site with the Classic UI.

I mean when people _verbally_ say "classic", they mean "Classic UI", because the latter can be a mouthful. In code and writing, we should use more care.

So let's call it buy its proper name of `classic-ui`. `classic` is a lazy shorthand. Even though it is only an internal identifier, the identifier appears in the docs.

**Comment by davisagli** [comment]
I still disagree. The distribution sets up the classic configuration of Plone in both backend and frontend aspects; it's not only about the UI.

**Comment by stevepiercy** [comment]
OK, then let's call it anything but "classic", because now you bring in "frontend" which also means "Volto" or "Classic UI".

Any of these names are more accurate. If none are satisfactory, we can look up terms in a synonym reference.

- plone
- configuration
- plone-configuration
- base
- core

**Comment by davisagli** [comment]
> OK, then let's call it anything but "classic", because now you bring in "frontend" which also means "Volto" or "Classic UI".]

Indeed. Using this distribution is making a choice about both the backend and frontend. It sets both of them up in the classic way. This is why classic is a fitting name.

> plone
> configuration
> plone-configuration

These are way too general and say nothing useful about what this distribution is.

> base
> core

It is NOT just the core, it is the core configured with the Classic UI and with related backend choices such as enabling portlets and the Folder content type.

**Comment by davisagli** [comment]
@stevepiercy I think we should probably have a real-time conversation about this, since we seem to be speaking past each other. I'm not available today.

**Comment by stevepiercy** [comment]
The word "classic" by itself is a problem. "classic way" is also a loaded term. TBH, the choice of "Classic UI" was also a mistake that we're now stuck with. "classic" is associated with "Classic UI", which excludes "Volto".

That's the problem.

I'm trying to come up with alternatives that do not use the already misused and abused word or term "classic".

- basic
- starter
- essential

**Comment by davisagli** [comment]
Can you explain how "classic" and "classic way" are problems? This is what we decided as a community to call the way that Plone sites were built before Volto was created, so your strong reaction here is catching me by surprise. I get how calling the Volto distribution "default" is loaded and I agree we should change that. I don't get why "classic" is a problem. To me it has positive connections: it's proven, stable, etc.

All your alternative proposals seem to be missing the mark to me so far. Classic Plone isn't basic or starter or essential; you can build a full, complex site with it just like you can with Volto.

**Comment by davisagli** [comment]
@stevepiercy Because both distributions are Plone and we're trying to make a distinction here.

**Comment by davisagli** [comment]
@stevepiercy I'm going to revise my position and say I don't have any big objection to changing the distribution name from "classic" to "classic-ui". The different customizations made in both built-in distributions are primarily about supporting one of the 2 frontends, so it does make some sense to give them names based on the frontend names. Earlier I was thinking it might be better to have the distributions named something else to reduce confusion, but it clearly doesn't, and that's inconsistent with what how we're planning to rename `default` to `volto` in plone.volto. So sure, I guess we can name the distributions after the frontends and say "the Classic UI distribution" or "the Volto distribution" when we need to clarify.

**Comment by stevepiercy** [comment]
@davisagli thank you for your patience with me. I was struggling with understanding what is and is not included. I'll work on this issue, following your example on the Volto one. As a bonus, I'll learn some details about how to create or modify a distribution.
